### ISSUE TITLE
[Feature Request]: Feat:adding a new page to display all contributors✨

### ISSUE THREAD

**Issue by trahulprabhu38** [Swarnendu0123/adopt-pet#21]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Feature Description

I want to add a page to showcase all the contributors of this website. @Swarnendu0123 can you assign me this issue.

### Use Case

-

### Priority

High

### Record

- [X] I agree to follow this project's Code of Conduct
- [X] I'm a GSSOC'24 Extd contributor
- [X] I want to work on this issue
- [X] I'm willing to provide further clarification or assistance if needed.

**Comment by Rohan1786** [comment]
**Hello sir this is rohan here**  i created this contributors page Now I have to Add This one into in your repository Do I add routing to page Can I using Browser Router
![Contributors](https://github.com/user-attachments/assets/ccc41cb6-66ef-4930-b61a-86ebdc572c1d)

**Comment by swrno** [comment]
Is it automated?

**Comment by Rohan1786** [comment]
yes  while using axios it will detect and fetch the contributions over this project i would like to request you just go through that page

**Comment by Rohan1786** [comment]
please update my pr i not received pr

**Comment by R1MO9** [comment]
@Rohan1786 Can you please assign this to me i will definitely provide a responsive and attractive ui
